packages/angular/build/src/tools/vite/middlewares/component-middleware.ts

```diff
@@ -18,7 +18,7 @@ export function createAngularComponentMiddleware(
       return;
     }
 
-    if (!req.url.startsWith(ANGULAR_COMPONENT_PREFIX)) {
+    if (!req.url.includes(ANGULAR_COMPONENT_PREFIX)) {
```

Comment:
It's unlikely but should we consider using `const pathname = pathnameWithoutBasePath(req.url, server.config.base);` to avoid false positives?

Reply:
good idea.